README.md

```diff
@@ -1,27 +1,43 @@
-# Project Name
+# FCC SURVEY FORM
 
-> One paragraph statement about the project.
+> A survey form used for generating data.
 
 ![screenshot](./app_screenshot.png)
```

Comment:
you can update this ,as what you have currently is better than what's shown here